.env

```diff
@@ -13,6 +13,15 @@ ECOMMERCE_TRANSACTIONS_PORT=8080
 ECOMMERCE_PAYMENT_METHODS_COMMIT_SHA=main
 ECOMMERCE_PAYMENT_METHODS_PORT=8083
 
+PAYMENT_TRANSACTION_GATEWAY_COMMIT_SHA=develop
+PAYMENT_TRANSACTION_GATEWAY_PORT=8092
```

Comment:
Change this port since already used by pdv mock

Reply:
> Change this port since already used by pdv mock

Ping @infantesimone 🙂